src/train_model.py

```diff
@@ -85,18 +85,18 @@ def train_model(config: Struct):
         arg_table.append(row)
     print(tabulate(arg_table, tablefmt="grid"))
 
-    # Print model info
-    print("\nModel Summary:")
-    total_params = model_summary(
-        model.to(config.device),
-        input_data=torch.ones(1, config.seq_len).long().to(config.device)).total_params
+    # Get number of parameters in model
+    model_parameters = filter(lambda p: p.requires_grad, model.parameters())
+    total_params = sum([np.prod(p.size()) for p in model_parameters])
 
     # Create unique label for model (model type, parameter count,
     # **hyperparameters, timestamp)
-    model_label = f"{config.model_type}_{total_params}" + \
-        f"_LR{config.learning_rate}_ED{config.embed_dim}" + \
-        f"_FFN{config.ffn_dim}_H{config.heads}_S{config.seq_len}" + \
-        f"_{datetime.now().strftime('%Y-%m-%d-%H:%M:%S')}"
+    if config.model_label is not None:
+        model_label = config.model_label
+    else:
+        model_label = f"{config.model_type}_{total_params}" + \
+            f"_LR{config.learning_rate}_ED{config.embed_dim}" + \
+            f"_FFN{config.ffn_dim}_H{config.heads}_S{config.seq_len}"
```

Comment:
Do we want to remove the time at the end (`datetime.now().strftime('%Y-%m-%d-%H:%M:%S')`)?

Reply:
Yes, because that's what it making it unique. It's nice until the requeuing doesn't work, because model_label will have a unique timestamp everytime the script is run.